tools/packaging/packaging.psm1

```
@@ -504,6 +504,29 @@ function Start-PSPackage {
                     }
                 }
             }
+            'osxpkg' {
+                $HostArchitecture = "x86_64"
+                if ($MacOSRuntime -match "-arm64") {
+                    $HostArchitecture = "arm64"
+                }
+                Write-Verbose "HostArchitecture = $HostArchitecture" -Verbose
+
+                $Arguments = @{
+                    Type = 'osxpkg'
+                    PackageSourcePath = $Source
+                    Name = $Name
+                    Version = $Version
+                    Force = $Force
+                    NoSudo = $NoSudo
+                    LTS = $LTS
+                    HostArchitecture = $HostArchitecture
+                }
+
+
+                if ($PSCmdlet.ShouldProcess("Create macOS Package")) {
+                    New-UnixPackage @Arguments
+                }
+            }
             default {
```

is there any package type that doesn't have an explicit statement?  If not, we can replace default with a throw that this type is unknown

---

I'm not sure if there are any other package types, as I have no much experience on this code base.  I'm going to let the default case throw an error that this type is unknown if you (or any other maintainers) ask!

---

@TravisEz13 anything you want @dahlia to change? If not, I will merge the PR as is.

---

I guess it was a question for MSFT team about release process :-)